doc/services/settings/index.rst

```diff
@@ -87,19 +87,19 @@ Zephyr has three storage backends: a Flash Circular Buffer
 (:kconfig:option:`CONFIG_SETTINGS_NVS`).
 
 You can declare multiple sources for settings; settings from
-all of these are restored when ``settings_load()`` is called.
+all of these are restored when :c:func:`settings_load()` is called.
 
 There can be only one target for writing settings; this is where
-data is stored when you call ``settings_save()``, or ``settings_save_one()``.
+data is stored when you call :c:func:`settings_save()`, or :c:func:`settings_save_one()`.
 
-FCB read target is registered using ``settings_fcb_src()``, and write target
-using ``settings_fcb_dst()``. As a side-effect,  ``settings_fcb_src()``
+FCB read target is registered using :c:func:`settings_fcb_src()`, and write target
+using :c:func:`settings_fcb_dst()`. As a side-effect,  :c:func:`settings_fcb_src()`
 initializes the FCB area, so it must be called before calling
-``settings_fcb_dst()``. File read target is registered using
-``settings_file_src()``, and write target by using ``settings_file_dst()``.
+:c:func:`settings_fcb_dst()`. File read target is registered using
+:c:func:`settings_file_src()`, and write target by using :c:func:`settings_file_dst()`.
 Non-volatile storage read target is registered using
-``settings_nvs_src()``, and write target by using
-``settings_nvs_dst()``.
+:c:func:`settings_nvs_src()`, and write target by using
+:c:func:`settings_nvs_dst()`.
```

Comment:
Folks more familiar than myself might want to double check whether this change is appropriate. Doesn't look like these are actually documented as actual API  so the change doesn't really turn them into a hyperlink. Not sure if the reason is something missing on the header file or a shortcoming of the API that doesn't really expose these functions as a "clean" API?

Reply:
AFAIK those are backend/internal APIs, they are not public APIs per se

Reply:
Right. I guess they are still technically C functions so it's not really incorrect to tag them as `c:func` anyway :)